scalasql/src/dialects/Dialect.scala

```diff
@@ -62,6 +62,13 @@ trait Dialect extends DialectTypeMappers {
     def put(r: PreparedStatement, idx: Int, v: Long) = r.setLong(idx, v)
   }
 
+  implicit def FloatType: TypeMapper[Float] = new FloatType
+  class FloatType extends TypeMapper[Float] {
+    def jdbcType = JDBCType.FLOAT
+    def get(r: ResultSet, idx: Int) = r.getFloat(idx)
+    def put(r: PreparedStatement, idx: Int, v: Float) = r.setDouble(idx, v)
```

Comment:
@rmorales-iaa should this be `setFloat`, for symmetry with the `getFloat` above?